scaleway/resource_k8s_pool_beta.go

```diff
@@ -150,6 +194,13 @@ func resourceScalewayK8SPoolBetaCreate(d *schema.ResourceData, m interface{}) er
 
 	d.SetId(newRegionalId(region, res.ID))
 
+	if d.Get("wait_for_pool_ready").(bool) { // wait for nodes to be ready if specified
```

Comment:
`// wait for the pool to be ready if specified` ? 

Either change comment or change name of the field.

Reply:
pool ready implies nodes ready, but would be nice to explain it indeed

Reply:
how about 
```
// wait for the pool to be ready if specified (including all its nodes)
```

Reply:
modified